apis/core.oam.dev/v1alpha1/policy_types.go

```diff
@@ -77,3 +79,11 @@ func (in SharedResourcePolicySpec) FindStrategy(manifest *unstructured.Unstructu
 	}
 	return false
 }
+
+// ReplicationPolicySpec defines the spec of replication policy
+// Override policy should be used together with replication policy to select the deployment target components
+type ReplicationPolicySpec struct {
+	Keys []string `json:"keys,omitempty"`
+	// Selector is the subset of selected components which will be replicated.
+	Selector []string `json:"selector,omitempty"`
```

Comment:
does the selector can only specify components?

Reply:
Currently yes. It works similar to the `selector` field in override policy. We can make further extension in the future.

Reply:
Yes, what else? Trait?

Reply:
Maybe other component selection ways, like selecting components through regex?

Reply:
But it's a string, it lacks flexible?

Reply:
Yes, it lacks flexibility. The consideration here is:
1. It aligns with the design of `override` policy, although `override` policy also lacks flexibility, but it is easy to understand.
2. It satisfy the scenario we currently encountered by far.
3. We could have more extension ways in the future. For now although it would be nice to make it more extensible, I do not have too much good points due to the lack of more advanced feature requests.